### ISSUE TITLE
Generalize Fusion 5 Survival Guide [Feedback]

### ISSUE THREAD

**Issue by MarcusSorealheis** [lucidworks/fusion-cloud-native#37]
NodePool naming: it seems that concept is used by GCP to define a set of node having the same definitions (for ressources) I've tried to check that thing in kubernetes before figuring out it wasn't a kube thing. while I understand the documentation was written with gcp in my I think it would benefit to avoid or anotate the references made to specific cloud providers to distinguish the kubernetes knowledge and some contextualized topic for GCP, AWS and other clouds

**Comment by thelabdude** [comment]
Node pools are not just a Google thing though, more of a cloud feature as GKE, AKS, and EKS all support them. You can still apply the concept of workload isolation to on-prem by using labels on nodes and `nodeSelector` labels on pods; you can have different nodes with different resources and assign pods to them as we documented. Customers will be responsible for defining their own nodes and attaching labels to them.

In other words, all the concepts in that section in the guide still apply, but instead of defining a NodePool, operators can attach labels to specific nodes and then assign pods by `nodeSelector`.
